lib/utils/fs.ts

```diff
@@ -17,7 +25,20 @@ function getConfigFromFile(fileName: string = 'package.json') {
       return config.pwmetrics || {};
     else return config;
   } else throw new Error(`Invalid config from ${fileName}`);
-  
+
 }
 
-module.exports = { getConfigFromFile };
+export function writeToDisk(fileName: string, data: string) {
+  return new Promise(async (resolve, reject) => {
```

Comment:
I meant do we need Promise here at all?
We can write inside function just
```
try { 
  const filePath = path.join(process.cwd(), fileName);
  await promisify(fs.writeFile)(filePath, data);
  logger.log(getMessageWithPrefix('SUCCESS', 'SAVED_TO_JSON', filePath));
} catch (err) {
      throw (err);
    }
```

Reply:
Oh, ok it makes sense you're right. The promise is removed there but the whole thing has to be awaited back to the call of `outputData` line 108 of `lib/index.ts` though.